tests/integration/user/user_create_test.go

```diff
@@ -1,41 +1,42 @@
 //go:build integration
```

Comment:
can you please implement a user_complete_test.go? its one which tests the whole "chain" and its much more efficient because it does not keep recreating users for example. See PHP please, and lets do the same for identifier

Reply:
done